### PR TITLE
Donate appeal

### PR DESCRIPTION
**Pull request by micahilbery**
Default:
Screenshot from 2022-06-24 14-08-55

On Hover:
Screenshot from 2022-06-24 14-09-03

### CODE REVIEW

**Review by wsmwk — APPROVED** [review]

File: website/thunderbird/102.0/whatsnew/index.html [hunk 1]
```diff
@@ -13,15 +13,14 @@ <h1 class="font-4xl md:font-hero font-thin md:mt-10 mb-8">{{ self.page_title() }
...
-      </p>
 
-      <div class="flex mb-20">
-        <a href="{{ donate_url('home_page_body') }}" class="btn-donate btn-donate-whatsnew font-lg">
-          <span class="inline-block w-4 mr-1">{{ svg('heart') }}</span> {{_('Donate')}}
-        </a>
-      </div>
+      <a href="{{ donate_url('home_page_body') }}" class="donation-appeal">
+        <div class="inline-block w-16 mr-4 text-donation">{{ svg('heart') }}</div>
+        <p class="tracking-wider leading-loose m-0">
```
Comment (Alecaddd) [inline website/thunderbird/102.0/whatsnew/index.html:19]: I'd use a slightly tighter kerning and line height.
```suggestion
        <p class="tracking-wide leading-normal m-0">
```

**Review by Alecaddd — APPROVED** [review]
Looks good, thanks

**Comment by Sancus** [comment]
Note that putting those `<strong>`tags in will bust localizations until localizers fix it. So I'm not sure it's a good idea to do that a few days before release...

**Comment by Sancus** [comment]
I'm going to merge this and then take the `<strong>` tags out and add them back later.

**Comment by ryanleesipes** [comment]
Thank you @Sancus !

**Pull request merged**